captum/attr/_utils/visualization.py

```diff
@@ -543,6 +546,24 @@ def visualize_text(datarecords: Iterable[VisualizationDataRecord]) -> None:
             )
         )
 
+    if legend:
+      dom.append('<div style="border-top: 1px solid; margin-top: 5px; \
+        padding-top: 5px; display: inline-block">'
+      )
+      dom.append('<b>Legend: </b>')
+
+      for value, label in zip(
+          [-1, 0, 1], ["Negative", "Neutral", "Positive Attribution"]
```

Comment:
Thank you for working on this PR @heytitle ! It feels a little bit that Attribution is related to positive only. Perhaps leaving it just `["Negative", "Neutral", "Positive"]` would be better and the fact that is attribution is intuitive ?